internal/olm/operator/registry/index/registry_pod.go

```diff
@@ -100,8 +100,6 @@ func (rp *RegistryPod) init(cfg *operator.Configuration) error {
 	}
 	rp.cfg = cfg
 
-	// TODO(rashmi/venkat): If FBCPath is empty, assign it a default value (maybe something thats hosted on a registry)
-
 	// validate the RegistryPod struct and ensure required fields are set
 	if err := rp.validate(); err != nil {
```

Comment:
I don't see the FBCpath being empty check in `rp.Validate`. It's probably removed accidentally.

Reply:
This may be an old version, since I don't have this part on my current branch.

We are trying to do away from the concept of paths, right? The only data points that we need are 
```
FBCcontent string
FBCdir     string
FBCfile    string
```

which is part of the `RegistryPod` struct. So we wouldn't have any field like FBCPath. Correct me if I'm wrong.

Reply:
The values of FBCDir and FBCfile only matter for the ephemeral pod, since that's where the contents are created.

Reply:
You're right. We're not using FBCPath anymore. We're good.